WISE-Community/WISE
# Site: Re-organized app directory structure; fixes #1077

How to test:

- Build and run the site app:
`npm run watch-all-site`
- Check to see that tests pass
- Go to http://localhost:8080/wise/site/index.html
- Select the different header links (about, features, news) and make sure the routes render properly
- Go to http://localhost:8080/wise and log in as a student
- Reload http://localhost:8080/wise/site/index.html
- You should now see the user account menu (person avatar) on the header
- Open the user account menu and select 'My Account'
- Make sure you see the student home page with a list of runs
